nixos/modules/services/web-apps/hitide.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.hitide;
+in
+{
+  options.services.hitide = {
+    enable = mkEnableOption "the hitide service";
+
+    package = mkOption {
```

Comment:
There is only one version of `hitide` available, and there are no variants on it. This option provides no value, in my opinion.

Reply:
I disagree... I started contributing to lotide/hitide development and would like to run development versions of it on my server, where I overwrite the package to build from different sources. This option gives me the abiltity to easily deploy different versions.
I would like to keep it therefore.

Reply:
Sure, it is fine to keep... but it sounds like what you really want is an overlay 🤔

Reply:
After some more thinking, I guess you're right here.